source/includes/gcp/_list_firewall_rules.md

```diff
@@ -0,0 +1,156 @@
+      "recordCount":4
+   }
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/instances</code>
+
+Retrieve a list of all instances in a given [environment](#administration-environments)
+
+Attributes | &nbsp;
+------- | -----------
+`IPProtocol`<br/>*string* | The IP protocol to which this rule applies. The protocol type is required when creating a firewall rule. This value can either be one of the following well known protocol strings (tcp, udp, icmp, esp, ah, ipip, sctp) or the IP protocol number.
+`creationTimestamp`<br/>*string* | Creation timestamp in RFC3339 text format.
+`description`<br/>*string* | An optional description of this resource. Provide this field when you create the resource.
+`direction`<br/>*string* | Direction of traffic to which this firewall applies, either INGRESS or EGRESS. The default is INGRESS. For INGRESS traffic, you cannot specify the destinationRanges field, and for EGRESS traffic, you cannot specify the sourceRanges or sourceTags fields.
+`disabled`<br/>*boolean* | Denotes whether the firewall rule is disabled. When set to true, the firewall rule is not enforced and the network behaves as if it did not exist. If this is unspecified, the firewall rule will be enabled.
+`id`<br/>*string* | [Output Only] The unique identifier for the resource. This identifier is defined by the server.
+`kind`<br/>*string* | Type of the resource.
+`logConfig`<br/>*FirewallLogConfig* | This field denotes the logging options for a particular firewall rule. If logging is enabled, logs will be exported to Stackdriver.
+`name`<br/>*string* | Name of the resource; provided by the client when the resource is created. The name must be 1-63 characters long, and comply with RFC1035. Specifically, the name must be 1-63 characters long and match the regular expression `a-z?. The first character must be a lowercase letter, and all following characters (except for the last character) must be a dash, lowercase letter, or digit. The last character must be a lowercase letter or digit.
+`network`<br/>*string* | URL of the network resource for this firewall rule. If not specified when creating a firewall rule, the default network is used.
+`priority`<br/>*integer* | Priority for this rule. This is an integer between 0 and 65535, both inclusive. The default value is 1000. Relative priorities determine which rule takes effect if multiple rules apply. Lower values indicate higher priority. For example, a rule with priority 0 has higher precedence than a rule with priority 1. DENY rules take precedence over ALLOW rules if they have equal priority. Note that VPC networks have implied rules with a priority of 65535. To avoid conflicts with the implied rules, use a priority number less than 65535.
+`selfLink`<br/>*string* | Server-defined URL for this resource.
+`sourceRanges`<br/>*list<string>* | [Output Only] Server-defined URL for the resource.
```

Comment:
We usually say *Array[string]* for a list of strings.
Remove `[Output Only]`, this is from the GCP doc for their REST API.